dnsimple/zone_distributions.go

```diff
@@ -0,0 +1,57 @@
+package dnsimple
+
+import "fmt"
+
+// ZoneDistribution is the result of the zone distribution check.
+type ZoneDistribution struct {
```

Comment:
How do you feel about renaming this struct into `DistributionResult`, and having both zoneDistributionResponse and zoneRecordDistributionResponse to use the same struct?

We don't _pay for structs_, but I hardly see the reason ATM to use 2 different types. It may makes things harder on the consumer side (especially given they are not an interface, and Go is strictly typed).

Reply:
I intentionally didn't declared an unified type because it looked like _premature DRY_.

On the other hand, we unify a lot of responses of the same type. Eg. registrar and domain services use `domainResponse `. So I think we can unify this as well.